src/components/molecules/ImageInput/ImageInput.tsx

```diff
@@ -117,7 +119,7 @@ export const ImageInput: React.FC<ImageInputProps> = ({
       </div>
       {isInputHidden && (
         <Button onClick={onButtonClick} variant="primary">
-          Upload banner
+          {btnLabel}
         </Button>
```

Comment:
In order to transition the old Button that is heavily modified by global styles, any opportunity to replace Button with ButtonNG that doesn't take too much time to deal with is a plus

```suggestion
        <ButtonNG onClick={onButtonClick} variant="primary">
          {text}
        </ButtonNG>
```

Reply:
fixed 65d922a